convert2rhel/utils.py

```diff
@@ -506,6 +506,23 @@ def string_to_version(verstring):
     return (epoch, version, release)
 
 
+def convert_to_int_or_zero(s):
```

Comment:
@r0x0d, you can remove this function and the associated unit test. It's not used anywhere anymore.

Reply:
done.